shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -2430,7 +2430,11 @@ - (BOOL)isPresentingViewController {
   return self.presentedViewController != nil || self.isPresentingViewControllerAnimating;
 }
 
-- (flutter::PointerData)generatePointerDataAtLastMouseLocation API_AVAILABLE(ios(13.4)) {
+- (flutter::PointerData)updateMousePointerDataFrom:(UIGestureRecognizer*)gestureRecognizer
+    API_AVAILABLE(ios(13.4)) {
+  CGPoint location = [gestureRecognizer locationInView:self.view];
+  CGFloat scale = [self flutterScreenIfViewLoaded].scale;
+  _mouseState.location = {location.x * scale, location.y * scale};
```

Comment:
nit: since you're calculating the x and y you could put them in local variables and use it a few lines below when setting `pointer_data.physical_x` and `y`.